pandora-common/src/networking/directory_client.ts

```diff
@@ -101,6 +103,13 @@ export const ACCOUNT_SETTINGS_DEFAULT = Object.freeze<IDirectoryAccountSettings>
 	interfaceChatroomOfflineCharacterFilter: 'ghost',
 });
 
+export const ACCOUNT_SETTINGS_LIMITED_LIMITS = Object.freeze({
+	displayName: TimeSpanMs(1, 'weeks'),
+} as const satisfies Partial<Record<keyof IDirectoryAccountSettings, number>>);
+
+export type DirectoryAccountSettingsLimitedKeys = keyof typeof ACCOUNT_SETTINGS_LIMITED_LIMITS;
+export type DirectoryAccountSettingsCooldowns = Partial<Record<DirectoryAccountSettingsLimitedKeys, number>>;
```

Comment:
These should be schemas to be validate-able:
```suggestion
export const DirectoryAccountSettingsLimitedKeysSchema = z.enum(ParseArrayNotEmpty(KnownObject.keys(ACCOUNT_SETTINGS_LIMITED_LIMITS)));
export type DirectoryAccountSettingsLimitedKeys = z.infer<typeof DirectoryAccountSettingsLimitedKeysSchema>;

export const DirectoryAccountSettingsCooldownsSchema = z.record(DirectoryAccountSettingsLimitedKeysSchema, z.number().optional());
export type DirectoryAccountSettingsCooldowns = z.infer<typeof DirectoryAccountSettingsCooldownsSchema>;
```